migration/migration_blackbox_test.go

```diff
@@ -209,6 +211,26 @@ func testMigration50(t *testing.T) {
 	assert.Nil(t, runSQLscript(sqlDB, "050-users-add-column-company.sql"))
 }
```

Comment:
this works, thanks for your time @hectorj2f .
https://github.com/almighty/almighty-core/pull/1160/commits/279a17eefa9a754f1182e215e445db14088e150e